lib/config.js

```diff
@@ -318,88 +320,41 @@ config.serverJobsAbandonedTimeoutSec = 30;
 config.attachedFilesDialogEnabled = true;
 config.devMode = (process.env.NODE_ENV ?? 'development') === 'development';
 
-// This will be populated by `lib/aws.js` later
-config.awsServiceGlobalOptions = {};
-
-const azure = {
-  // Required
-  azureIdentityMetadata:
-    'https://login.microsoftonline.com/common/.well-known/openid-configuration',
-  // azureIdentityMetadata: 'https://login.microsoftonline.com/<tenant_name>.onmicrosoft.com/.well-known/openid-configuration',
-  // or equivalently: 'https://login.microsoftonline.com/<tenant_guid>/.well-known/openid-configuration'
-  //
-  // or you can use the common endpoint
-  // 'https://login.microsoftonline.com/common/.well-known/openid-configuration'
-  // To use the common endpoint, you have to either set `validateIssuer` to false, or provide the `issuer` value.
-
-  // Required, the client ID of your app in AAD
-  azureClientID: '<your_client_id>',
-
-  // Required, must be 'code', 'code id_token', 'id_token code' or 'id_token'
-  azureResponseType: 'code id_token',
-
-  // Required
-  azureResponseMode: 'form_post',
-
-  // Required, the reply URL registered in AAD for your app
-  azureRedirectUrl: 'http://localhost:3000/auth/openid/return',
-
-  // Required if we use http for redirectUrl
-  azureAllowHttpForRedirectUrl: false,
-
-  // Required if `responseType` is 'code', 'id_token code' or 'code id_token'.
-  // If app key contains '\', replace it with '\\'.
-  azureClientSecret: '<your_client_secret>',
-
-  // Required to set to false if you don't want to validate issuer
-  azureValidateIssuer: false,
-
-  // Required if you want to provide the issuer(s) you want to validate instead of using the issuer from metadata
-  azureIssuer: null,
+/** The client ID of your app in AAD; required. */
+config.azureClientID = '<your_client_id>';
```

Comment:
We'll have to remember to reconfigure our existing prod setups (for all envs).

Reply:
What exactly do you think we'll have to change configuration for? I only removed unused options; I didn't change any of the ones we're using in existing configs (unless I changed something else on accident).

Reply:
Oh sorry! I misread the config changes and thought there was a change in the level of nesting for the Azure options. But this all looks fine!